src/body.js

```diff
@@ -380,6 +413,10 @@ export function writeToStream(dest, instance) {
 		// body is string
 		dest.write(body);
 		dest.end();
+	} else if (isURLSearchParams(body)) {
+		// body is URLSearchParams
+		dest.write(new Buffer(String(body)));
```

Comment:
So I was actually going to submit another PR after this to replace all of the new Buffers... I didn't want to mix concerns, hehe. But yes, I understand now... don't do bad stuff in the new code :p